test/test_helper.rb

```diff
@@ -1,3 +1,13 @@
+# Coverage - Keep these two lines at the top of this file
+require 'simplecov'
+require 'coveralls'
+SimpleCov.formatters = [SimpleCov::Formatter::HTMLFormatter, Coveralls::SimpleCov::Formatter]
+SimpleCov.start do
+  minimum_coverage 52
+  refuse_coverage_drop
+  track_files "{lib,test}/**/*.rb"
```

Comment:
Well, it is right now. I guess that's checking that we're exercising all of our test code. I'm willing to remove the test directory from coverage if you think that would be better.